resources/assets/components/InboxItem/index.js

```diff
@@ -22,8 +22,10 @@ class InboxItem extends React.Component {
     const post_id = post['id'];
     const signup = this.props.details.signup;
 
+    var posts = signup.posts ? signup.posts : post.siblings.data;
```

Comment:
Hmm, this feels a little awkward with the implicit dependency on the `details` prop. I don't have any great suggestions off the top of my head, but perhaps something we can simplify a bit.